src/risclog/sqlalchemy/cache.py

```diff
@@ -0,0 +1,495 @@
+from risclog.sqlalchemy.model import ObjectBase
```

Comment:
This module is only covered by ~77%. Would it be possible, to add at least tests for 
```
_get_model_cache
get_or_create
_assign_sequences
create
```
It seems, that actually some caching code is not tested here. 

The `check_memory_usage` could be ignored, if it is really used but too much work for now.